__tests__/app.test.js

```diff
@@ -145,15 +144,37 @@ describe("/api/articles/:article_id/comments", () => {
       .get("/api/articles/lizzie/comments")
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe("Sorry, try again!")
+        expect(body.msg).toBe("Sorry - not found!")
       })
     })
     test("404: throws error when article id is correct type but not on the table", () => {
       return request(app)
       .get("/api/articles/500/comments")
       .expect(404)
       .then(({ body }) => {
-        expect(body.err.msg).toBe("Sorry, I couldn't find that!")
+        expect(body.msg).toBe("Sorry, I couldn't find that article!")
       })
     })
-  })
\ No newline at end of file
+  })
+  describe("/api/articles/:article_id/comments POST", () => {
+    test("201: successfully inserts a comment", () => {
```

Comment:
This is a good start, but this test description should also include the behaviour of actually receiving the posted as well